requirements.txt

```diff
@@ -24,7 +24,8 @@ pyparsing==2.4.5
 einops==0.3.2
 pycocotools==2.0.4
 protobuf~=3.19.0
-deci-lab-client==2.38.0
+deci-common==3.4.1
+deci-lab-client==2.55.0b16158
```

Comment:
please use a released version

Reply:
yes, just waiting on a new version of the API to be released